src/main/java/com/cloudbees/groovy/cps/tool/Driver.java

```diff
@@ -78,18 +79,21 @@ public void run() throws Exception {
              *      2. has a Closure parameter in one of the arguments, not in the receiver
              */
             Predicate<ExecutableElement> selector = (e) -> {
-                return e.getKind() == ElementKind.METHOD
-                    && e.getModifiers().containsAll(PUBLIC_STATIC)
-                    && e.getParameters().subList(1, e.getParameters().size()).stream()
-                        .anyMatch(p -> t.types.isAssignable(p.asType(), closureType))
-                    && !EXCLUSIONS.contains(e.getEnclosingElement().getSimpleName().toString()+"."+e.getSimpleName().toString());
+                boolean r = e.getKind() == ElementKind.METHOD
+                        && e.getModifiers().containsAll(PUBLIC_STATIC)
```

Comment:
Ah-ha, yeah, that overload resolution would explain why my `.each(List,Closure)` and `.each(Iterable,Closure)` calling an intermediate `._each(T,Closure)` that uses `InvokerHelper.asIterator` to call `.each(Iterator,Closure)` works while this doesn't, right?